Backendless/Messaging/DeviceRegistrationDto.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
-using System.Collections.Generic;
 using Weborb.Service;
+using Plugin.DeviceInfo;

```

Comment:
Should there be a conditional compiler statement (#IF MOBILE for example) ?

Reply:
All other projects doesn't have reference to this class